.github/workflows/caching-build.yml

```diff
@@ -192,9 +117,15 @@ jobs:
             antistall: ['', '--disable-antistall']
           fail-fast: false
     steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/restore-kernel-cache
+        with:
+          git-repo: "https://git.kernel.org/pub/scm/linux/kernel/git/tj/sched_ext.git"
+          branch: for-6.13
```

Comment:
should branch also be a variable?

Reply:
A variable at the top of this workflow you mean? It is duplicated a lot which is kind of annoying and will get changed in the future, so could make sense. I'm not sure what the best way to do that is though - will have a think.